code/game/machinery/bots/secbot.dm

```diff
@@ -26,10 +26,13 @@
 	radio_frequency = SEC_FREQ //Security channel
 	bot_type = SEC_BOT
 	model = "Securitron"
+	var/movement_delay = 4
+
+	#define BOT_MOVEMENT_DELAY 4 // This will be the default movement delay
```

Comment:
that's not a proper use of define. You can't make a define owned by something, and anyway why even use it here?You can access the value you assigned to vars from coding with initial(varname), so for example initial(movement_delay) will always return 4 in this case